src/main/java/hudson/plugins/jacoco/JacocoPublisher.java

```diff
@@ -55,33 +68,54 @@
     /**
      * Variables containing the configuration set by the user.
      */
-	private final String execPattern;
-	private final String classPattern;
-	private final String sourcePattern;
-	private final String inclusionPattern;
-	private final String exclusionPattern;
-	
-    
-    private final String minimumInstructionCoverage;
-    private final String minimumBranchCoverage;
-    private final String minimumComplexityCoverage;
-    private final String minimumLineCoverage;
-    private final String minimumMethodCoverage;
-    private final String minimumClassCoverage;
-    private final String maximumInstructionCoverage;
-    private final String maximumBranchCoverage;
-    private final String maximumComplexityCoverage;
-    private final String maximumLineCoverage;
-    private final String maximumMethodCoverage;
-    private final String maximumClassCoverage;
-    private final boolean changeBuildStatus;
+    private String execPattern;
+    private String classPattern;
+    private String sourcePattern;
+    private String inclusionPattern;
+    private String exclusionPattern;
+
+    private String minimumInstructionCoverage;
+    private String minimumBranchCoverage;
+    private String minimumComplexityCoverage;
+    private String minimumLineCoverage;
+    private String minimumMethodCoverage;
+    private String minimumClassCoverage;
+    private String maximumInstructionCoverage;
+    private String maximumBranchCoverage;
+    private String maximumComplexityCoverage;
+    private String maximumLineCoverage;
+    private String maximumMethodCoverage;
+    private String maximumClassCoverage;
+    private boolean changeBuildStatus;
     
 	private static final String DIR_SEP = "\\s*,\\s*";
 
+    @DataBoundConstructor
+    public JacocoPublisher() {
+        this.execPattern = "**/**.exec";
```

Comment:
BTW if you want to avoid duplicating defaults with `config.jelly`, see [this trick](https://github.com/jenkinsci/workflow-plugin/blob/d9234cc89d6bf7245d0d1f71e96faf8d2ea99ee4/COMPATIBILITY.md#handling-default-values).